### PR TITLE
Ef 85 payment history add delete functionaility

### PR DESCRIPTION
**Pull request by ryanzhxu**

### CODE REVIEW

File: server/routes/tenants.js [hunk 1]
```diff
@@ -24,67 +30,74 @@ router.get('/tenants/:tenantId', async (req, res, next) => {
...
-// gets list of all tenants
-router.get('/tenants', async (req, res) => {
+router.post('/tenants', async (req, res) => {
+  const propertyId = req.body.propertyId;
+
+  if (!mongoose.isValidObjectId(propertyId)) {
+    res.status(StatusCodes.BAD_REQUEST).json({ error: 'Invalid propertyId format' });
+  }
+
+  const foundProperty = await Property.findById(propertyId);
```
[the remark this answers is not in the document]
Comment (ryanzhxu) [reply server/routes/tenants.js]: wrapped in try-catch clause.

File: server/models/tenant.js [hunk 2]
```diff
@@ -74,8 +73,7 @@ const tenantSchema = new mongoose.Schema(
       },
     ],
     propertyId: {
-      type: Schema.Types.ObjectId,
-      ref: 'Property',
+      type: String,
```
[the remark this answers is not in the document]
Comment (ryanzhxu) [reply server/models/tenant.js]: reverted.

File: server/models/database.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ const dotenv = require('dotenv');
 
 dotenv.config();
 
-const uri = process.env.NODE_ENV ? process.env.DB_HOST_REMOTE : process.env.DB_HOST;
+const uri = !process.env.NODE_ENV ? process.env.DB_HOST_REMOTE : process.env.DB_HOST;
```
[the remark this answers is not in the document]
Comment (ryanzhxu) [reply server/models/database.js]: reverted.

File: server/models/tenant.js [hunk 4]
```diff
@@ -1,5 +1,4 @@
 const mongoose = require('mongoose');
-const { Schema } = require('mongoose');
```
[the remark this answers is not in the document]
Comment (ryanzhxu) [reply server/models/tenant.js:2]: added back.

File: client/src/components/tenant/Tenant.css [hunk 5]
```diff
@@ -10,7 +10,7 @@
 .payment-history-container {
   max-height: 250px;
   overflow-y: auto;
-  margin-left: 14px;
```
Comment (hexxu) [inline client/src/components/tenant/Tenant.css:13]: I think the styling for the profile section might still need to be adjusted. Not too sure if this change happened earlier, but the profile image is no longer centered and the `Edit` text is not visible in the button
image

Comment (hexxu) [reply client/src/components/tenant/Tenant.css:13]: I guess it's not too big a deal now tho.

**Review by hexxu — APPROVED** [review]

**Pull request merged**